plugins/woocommerce/includes/wc-order-functions.php

```diff
@@ -610,7 +610,7 @@ function wc_create_refund( $args = array() ) {
 			}
 
 			// Trigger notification emails.
-			if ( ( $remaining_refund_amount - $args['amount'] ) > 0 || ( $order->has_free_item() && ( $remaining_refund_items - $refund_item_count ) > 0 ) ) {
+			if ( (bool) apply_filters( 'woocommerce_order_is_partially_refunded', ( $remaining_refund_amount - $args['amount'] ) > 0 || ( $order->has_free_item() && ( $remaining_refund_items - $refund_item_count ) > 0 ), $order->get_id(), $refund->get_id() ) ) {
```

Comment:
Please let's document the filter and give a hint at when it should/should not be used/what the implications are, and capture the three `@param` types: then I think this is good to go.

Reply:
Thanks for the review, @barryhughes! Just added the doc block.